tests/conftest.py

```diff
@@ -55,3 +55,52 @@ def convolutional_model(random_data):
     model.compile(optimizer="adam", loss="categorical_crossentropy")
 
     return model
+
+
+@pytest.fixture()
+def convolutional_model_for_vanilla_gradients(random_data):
```

Comment:
I think naming could be better with something that really explains what's the difference for vanilla gradients. Something like `convolutional_model_with_independant_activation_layer`